### ISSUE TITLE
16.1.1.0 crashing when attempting to access 3d map

### ISSUE THREAD

**Issue by Gopher-nz** [EDDiscovery/EDDiscovery#3411]
I have read the notes on 3d map crashing from 12 months ago. However, this problem is only happening on this version.

**Comment by Gopher-nz** [comment]
2d map works fine

**Comment by Eahlstan** [comment]
So you read it. But did you also do what's written in it?

**Comment by Gopher-nz** [comment]
Yup and still crashes

**Comment by Eahlstan** [comment]
Well, provide more info. "It crashes" is a nice statement, but absolutely useless when it comes to error handling.

**Comment by robbyxp1** [comment]
No changes to 3dmap in this version.

**Comment by Gopher-nz** [comment]
Sorry if you think my comment is absolutely useless.  Maybe you could also be more clearer too and let me know what more info you would like and how I get it.

**Comment by Gopher-nz** [comment]
> No changes to 3dmap in this version. Therefore there is no reason why 16.1 will perform any differenty to 16.0.

Why would 2d map work but not 3d map. Although I have not tried some of the other modules yet,  I suspect they will also work fine too.

**Comment by Gopher-nz** [comment]
Okay as far as error messages go - I get a small window pop up saying "EDDiscovery is not responding" and the entire EDDiscovery window fades. If there are any other areas I can get specific information to determine whether it is my computer or not I would appreciate someone letting me know where to look. So far I can add EDSM and Inara tabs with no problems. However, I also tried Local 3d map todfay and that didn't work either.

**Comment by Gopher-nz** [comment]
I have now Uninstalled it - deleted both instances of ED Discovery in Program Files and AppData. I reinstalled version 15.0.4 when the new 3d map was introduced and this works fine. I can access both the new and legacy map. The moment I update to the latest version the software stops working again.

**Comment by robbyxp1** [comment]
What we need is a screenshot of any error message produced when it crashes, the contents of the log folder in appdata\eddiscovery, ie. something which tells us what its upset with.

**Comment by robbyxp1** [comment]
And what is your hardware setup?  Graphics card, processor, memory,etc

**Comment by Gopher-nz** [comment]
Okay the image is problematic as I tried to do a screen grab earlier today but I could not get the error message to pop up when I was screen grabbing, best I could do was a print screen - dxdiag details attached too.
![R](https://user-images.githubusercontent.com/13024452/233835247-0e5804d0-bb7f-48fb-80fa-70b9e5e5ed5f.png)
[DxDiag.txt](https://github.com/EDDiscovery/EDDiscovery/files/11303458/DxDiag.txt)

**Comment by Gopher-nz** [comment]
Log file 
[Trace_2023-04-23-22-42-02.0.log](https://github.com/EDDiscovery/EDDiscovery/files/11303472/Trace_2023-04-23-22-42-02.0.log)

**Comment by robbyxp1** [comment]
In the elite launcher, under Options, there is display hardware survey. please send that so we can see what you have running.

The log shows it got stuck launching the opengl shaders:

[2023-04-23T10:42:25.435Z] GL Context 65537
[2023-04-23T10:42:25.435Z] Version  4.6.14800
[2023-04-23T10:42:25.435Z] Version  4.6.14800 Core Profile Context 21.50.20.01 30.0.15020.1001
[2023-04-23T10:42:25.435Z] Vendor  ATI Technologies Inc.
[2023-04-23T10:42:25.436Z] Shading lang 4.60
[2023-04-23T10:42:25.436Z] UBS=572657868
[2023-04-23T10:42:25.436Z] UB v15 f15 g15 tc15 te15
[2023-04-23T10:42:25.436Z] tex layers 2048 
[2023-04-23T10:42:25.436Z] Vertex attribs 29 
[2023-04-23T10:42:25.436Z] Shader storage buffer bindings 64 

Obv it does not like your graphics hardware.  Can we see it via the report above.

**Comment by Gopher-nz** [comment]
Sorry for taking a while to get back to you. I usually play using VR (and in case you are wondering, I peep under my headset to see the EDDiscovery Screen), which means the launcher screen is bypassed in VR mode. However, starting it in normal mode and opening up the hardware report from the launcher screen provides the following (PS - 3d maps does not work when I start the game in non-VR mode too):
[Hardware survey.txt](https://github.com/EDDiscovery/EDDiscovery/files/11305103/Hardware.survey.txt)

**Comment by klightspeed** [comment]
> [2023-04-23T10:42:25.435Z] Version 4.6.14800 Core Profile Context 21.50.20.01 30.0.15020.1001

It looks like you might be using old drivers.  A similar problem with the same driver version (30.0.15020.1001) was reported on the Adobe support forum: https://community.adobe.com/t5/photoshop-ecosystem-discussions/photoshop-keeps-crashing/td-p/13157768

**Comment by Gopher-nz** [comment]
I have gone to the AMD website and downloaded the most up to date drivers again and it makes no difference. Same issue. 
[Trace_2023-04-24-12-22-12.0.log](https://github.com/EDDiscovery/EDDiscovery/files/11305375/Trace_2023-04-24-12-22-12.0.log)
[DxDiag v2.txt](https://github.com/EDDiscovery/EDDiscovery/files/11305376/DxDiag.v2.txt)
[Hardware survey V2.txt](https://github.com/EDDiscovery/EDDiscovery/files/11305378/Hardware.survey.V2.txt)

**Comment by klightspeed** [comment]
Whereas your previous trace stopped after:
```
[2023-04-23T10:42:25.436Z] Shader storage buffer bindings 64
```

Your new trace shows events continued to be processed for a few minutes after it appears two 3D map instances were initialized.

**Comment by Gopher-nz** [comment]
Just comparing the section on the latest Hardware suvrey log to the previous one I sent:
[2023-04-24T00:24:58.822Z] GL Context 65537
[2023-04-24T00:24:58.823Z] Version  4.6.0
[2023-04-24T00:24:58.823Z] Version  4.6.0 Compatibility Profile Context 23.4.2.230412
[2023-04-24T00:24:58.823Z] Vendor  ATI Technologies Inc.
[2023-04-24T00:24:58.823Z] Shading lang 4.60
[2023-04-24T00:24:58.823Z] UBS=4194304
[2023-04-24T00:24:58.823Z] UB v36 f36 g36 tc36 te36
[2023-04-24T00:24:58.823Z] tex layers 8192 
[2023-04-24T00:24:58.823Z] Vertex attribs 32 
[2023-04-24T00:24:58.823Z] Shader storage buffer bindings 32 

I don't understand this but I have to ask if the new driver has actually rolled back to a previous version?

**Comment by klightspeed** [comment]
The only thing I can find for 30.0.15020.1001 on the AMD site is release notes for Adrenalin 22.5.1 for RX 6700:
https://www.amd.com/en/support/kb/release-notes/rn-rad-win-22-5-1-rx6700

If installed through Windows Update, the latest version would probably show 31.0.14051.1000 (as detailed in https://www.amd.com/en/support/kb/release-notes/rn-rad-win-23-4-2)

**Comment by Gopher-nz** [comment]
Oh well, regardless, it has not solved the problem which is a shame. Because irrespective of feeling a bit stupid over all of this I just want to get it fixed. It is bad enough not knowing the source of the problem and even worse, having to bother devs or testers who have more important stuff to do.

**Comment by robbyxp1** [comment]
The only thing i can recomment is 1) keep your drivers up to date 2) pray that the graphics adaptor manufacturer actually implemented open GL 4.6 properly (I'm looking at you AMD).

Closing now.
